client/src/styles/Profile-styles/EditProfile.module.css

```diff
@@ -0,0 +1,68 @@
+.Edit__container {
+  height: 85vh;
+  padding: 4rem 2rem 0;
+}
+
+.Edit__heading {
+  border-bottom: 1px solid #c4c4c4;
+  margin-bottom: 5rem;
+}
+
+.Edit__button {
+  background-color: var(--clr-royal-blue);
+  box-shadow: 0 2rem 0 rgba(#000, 0.4);
```

Comment:
Browser devtools show 'invalid property value' flag for this rule

Reply:
Will fix it, nice catch.

Reply:
I said `rgba` but passed in `hex` values to the `box-shadow`, no wonder. I am so used to writing it like that using `sass` that I failed to pay attention that this is plain `css`.